pkg/kubelet/logs/container_log_manager.go

```diff
@@ -412,8 +412,9 @@ func (c *containerLogManager) rotateLatestLog(id, log string) error {
 			// This shouldn't happen.
 			// Report an error if this happens, because we will lose original
 			// log.
-			klog.Errorf("Failed to rename rotated log %q back to %q: %v, reopen container log error: %v", rotated, log, renameErr, err)
+			klog.ErrorS(renameErr, "Failed to rename rotated log %q back to %q: %v", "rotatedLog", rotated, "log", log)
```

Comment:
There are actually two errors here and we've dropped `err`. @serathius what should we do here?

Reply:
Looking into next line with return we just wrap the `renameErr`, based on that I think it's ok to report here only the renameErr.

Alternative would be to also change the return to create error that wraps both `err` and `renameErr` and then pass both to klog.ErrorS and return.